tenable/io/v3/vm/agent_config.py

```diff
@@ -0,0 +1,93 @@
+'''
+Agent Config
+============
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`agent config <agent-config>` API endpoints.
+
+Methods available on ``tio.agent_config``:
```

Comment:
This needs to be updated in accordance with sphinx.

Reply:
Done.